packages/mdc-form-field/foundation.js

```diff
@@ -37,7 +44,9 @@ export default class MDCFormFieldFoundation extends MDCFoundation {
 
   constructor(adapter) {
     super(Object.assign(MDCFormFieldFoundation.defaultAdapter, adapter));
-    this.clickHandler_ = (evt) => this.handleClick_(evt);
+
+    this.clickHandler_ = /** @private {!EventListener} */ (
```

Comment:
Does closure compiler let you do that? Put the `@private` with the type cast instead of before the field name?